File: Getting Started/IsANumberPrime.java
import java.util.*;

public class IsANumberPrime {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int t = scn.nextInt();

        // write ur code here
        for (int j = 0; j < t; j++) {
            int n = scn.nextInt();
            int flag = 0;
            if (n == 2) {
                System.out.println("prime");
            } else {
                for (int i = 2; i * i <= n; i++) {
                    if (n % i == 0) {
                        flag = 1;
                        break;
                    }
                }
                if (flag == 1) {
                    System.out.println("not prime");
                } else {
                    System.out.println("prime");
                }
            }
        }


    }
}

File: 1. Getting Started/cpp/IsANumberPrime.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
    int t;
    cin >> t;
    while(t--){
        int n,flag=0;
        cin >> n;
        for(int i=2;i<=sqrt(n);i++){
            if(n%i==0){
                flag=1;
                break;
            }
        }
        if(flag==1){
            cout << "not prime" << endl;
        }
        else{
            cout << "prime" << endl;
        }
    }
    return 0;

    //write your code here
    
}